app/helpers/hyrax/facets_helper.rb

```diff
@@ -0,0 +1,249 @@
+# frozen_string_literal: true
+
+module Hyrax
+  module FacetsHelper
+    # All methods in this module are from Blacklight::FacetsHelperBehavior, blacklight v6.24.0
+    # This module is used to ensure Hyrax facet views that rely on deprecated Blacklight helper methods are still functional
```

Comment:
Are all of these helpers required? If not, can the unneeded ones be left out?

Reply:
I removed the helper methods that are not relevant to this ticket.